src/AzSdk.test.reference.props

```diff
@@ -1,15 +1,14 @@
-<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-	<Import Project="$([MSBuild]::GetPathOfFileAbove('test.props'))" />	
-	
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">	
   <PropertyGroup>
 		<TargetFrameworks>netcoreapp2.0</TargetFrameworks>
 	</PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Test.HttpRecorder" Version="[1.13.3, 2.0.0)" />
     <PackageReference Include="Microsoft.Rest.ClientRuntime.Azure.TestFramework" Version="[1.7.6, 2.0.0)" />
     <PackageReference Include="Microsoft.Azure.ResourceManager" Version="[1.1.0-preview]" />
-    
-    <!-- This is needed for discovering tests in test explorer -->
-    <PackageReference Include="System.Runtime.InteropServices" Version="4.3.0" />	
+    <PackageReference Include="xunit" Version="2.3.1" />
+		<PackageReference Include="xunit.runner.visualstudio" Version="2.3.1" />
```

Comment:
nit: indention.

Reply:
Will fix the indentation.

Reply:
I checked and I don't know how I can fix the indentation, I see this locally
![image](https://user-images.githubusercontent.com/20732208/58834469-5545aa00-8608-11e9-9e75-19c566ceb117.png)